content/departments/cloud/index.md

```diff
@@ -66,30 +50,30 @@ Build a **fully managed platform** for using Sourcegraph that can **(by EOFY23)
 ## How to contact the team and ask for help
 
 - For emergencies and incidents, alert the team using Slack command `/genie alert [message] for cloud` and optionally tag the `@cloud-support` handle.
-- For internal Sourcegraph teammates, join us in #cloud slack channel to ask questions or request help from our team.
+- For internal Sourcegraph teammates, join us in #ask-cloud-ops slack channel to ask questions or request help from our team.
 - For [managed instance requests](#managed-instance-requests) or requests for help that requires action for the Cloud team engineers _(exp. coding, infrastructure change etc.)_ please create a GH issue and assign a `team/cloud` label. You can also post a follow up message on the #cloud slack channel
-- You may tag the `@cloud-support` handle if you are looking for immediate attention, and it will notify our [on-call engineers](#on-call). Please avoid tagging/DM a specific teammate or the `@cloud-team` handle, this is to try and protect their focus.
+- You may tag the `@cloud-support` handle if you are looking for immediate attention, and it will notify our [on-call engineers](#on-call). Please avoid tagging/DM a specific teammate or the `@cloud-team` handle.
 
-## Managed Instance
+## Cloud Instances
```

Comment:
This is an intentional terminology change? There are countless places we use "managed instances" and in reality most of the time people tend to type its abbreviation "MI", which is well-perceived. Where "CI" means... totally something different. @nelsonjr @rafax

Reply:
My impression was that we use "Cloud Instance" more often than "Managed Instance" now (vs 1-year ago when this was written), but I don't feel strongly about this change - if you think we should stay with MI, happy to keep that.

Reply:
A quick unscientific stats across our repos 😂 

<img width="596" alt="CleanShot 2023-07-05 at 09 30 25@2x" src="https://github.com/sourcegraph/handbook/assets/2946214/36de3706-d087-46e4-b026-347f1dd556ad">


<img width="609" alt="CleanShot 2023-07-05 at 09 31 01@2x" src="https://github.com/sourcegraph/handbook/assets/2946214/372f2648-0ef7-4df7-927d-c8d64e5eb4bb">

But I'm OK with any term, original intention was to make sure this ("Cloud Instances") is term  we are settling with, so whenever I see the another term, I know I should change/update them for consistency.